src/App.vue

```diff
@@ -31,13 +31,18 @@
 <script lang="ts">
 // Fix for breaking change introduced in polkadot js v7.x
 // https://polkadot.js.org/docs/api/FAQ/#since-upgrading-to-the-7x-series-typescript-augmentation-is-missing
+import 'reflect-metadata';
```

Comment:
How about moving this and `import '@polkadot/api-augment';` to `src/index.ts` since it's used globally?

Reply:
I don't see src/index.ts. Quasar is using multiple boot files to start the portal. 
![image](https://user-images.githubusercontent.com/8452361/174552810-d0ea4e16-2699-4b89-b9a0-e75d8807ffd3.png)
Maybe I can create a new boot file an put `import 'reflect-metadata'` there

Reply:
hmm, actually no. I thought there was a index.ts root file (like most programs), but I must have been mistaken.
Since App.vue is the root file, let's just keep it here.

Can I merge this PR?